pip/commands/list.py

```diff
@@ -53,7 +58,7 @@ def __init__(self, *args, **kw):
             help=('If in a virtualenv that has global access, do not list '
                   'globally-installed packages.'),
         )
-        self.cmd_opts.add_option(
+        cmd_opts.add_option(
             '--user',
```

Comment:
Not that I'm aware of. I must have removed it during one of my "let's clean things up" sprees. I noticed that this had `self` while others didn't. I'll revert it.